src/systemd/cockpit-wsinstance-http.service.in

```diff
@@ -2,11 +2,11 @@
 Description=Cockpit Web Service http instance
 Documentation=man:cockpit-ws(8)
 BindsTo=cockpit.service
-Requires=cockpit-session.socket cockpit-session-socket-user.service
-After=cockpit-session.socket cockpit-session-socket-user.service
```

Comment:
I actually sent a remark about that in your original PR, but then deleted it again. I am on the fence here, but I slightly leaned towards mentioning them explicitly as the unit directly uses that user (or at least the group). But no strong opinion, this is fine.